drivers/ethernet/eth_xilinx_axienet.c

```diff
@@ -5,6 +5,8 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
Could you modify the commit subject to `drivers: ethernet: xilinx: ....` so that it is clear what driver is being modified, now it looks like a generic change.